deployment/network-operator/README.md

```diff
@@ -428,17 +429,19 @@ imagePullSecrets:
 | `ofedDriver.upgradePolicy.drain.deleteEmptyDir`             | bool   | `false`                           | continue even if there are pods using emptyDir                                                                                                                                |
 | `ofedDriver.upgradePolicy.waitForCompletion.podSelector`    | string | not set                           | specifies a label selector for the pods to wait for completion before starting the driver upgrade                                                                             |
 | `ofedDriver.upgradePolicy.waitForCompletion.timeoutSeconds` | int    | not set                           | specify the length of time in seconds to wait before giving up for workload to finish, zero means infinite                                                                    |
+| `ofedDriver.containerResources`                 | []     | `[{"name": "mofed-container", "requests": {}, "limits": {}}]` | Optional [resource requests and limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for the container                                |
```

Comment:
i would keep that empty. and in description say: resource requirements for individual containers (or something of this sort)
then add separate section about container resources (and link to it from the type )

rinse and repeat for all relevant sections